lua/entities/gmod_wire_expression2/core/strfunc.lua

```diff
@@ -43,34 +40,36 @@ local function findFunc( self, funcname, typeids, typeids_str )
 		if not func then
 			for i=#typeids,1,-1 do
```

Comment:
Would also be nice if you could store ``#typeids`` in a variable so it isn't used two times more than necessary.